apps/backend/src/app/modules/maps/maps.service.ts

```diff
@@ -760,7 +760,7 @@ export class MapsService {
     );
   }
 
-  private async checkCreateDto(userID: number, dto: CreateMapDto) {
+  async checkCreateDto(userID: number, dto: CreateMapDto) {
```

Comment:
Above tests being E2E means you shouldn't need this, in general don't like changing the access modifiers in service logic just so tests can access it!